sci-electronics/splat/splat-1.4.2.ebuild

```diff
@@ -44,17 +46,17 @@ src_configure() {
 src_compile() {
 	local CC=$(tc-getCC) CXX=$(tc-getCXX)
 
-	${CXX} -Wall ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat -lm -lbz2 || die
+	${CXX} -Wall ${CXXFLAGS} ${LDFLAGS} itwom3.0.cpp splat.cpp -o rfsplat -lm -lbz2
```

Comment:
CPPFLAGS

Reply:
Why drop the || die?

Reply:
hmm for some reason I thought we don't need to die, guess I mixed it with `emake`